CODEOWNERS

```diff
@@ -6,22 +6,22 @@
 .known-issues/*                          @inakypg @nashif
 arch/arc/                                @vonhust @ruuddw
 arch/arm/                                @MaureenHelm @galak
-arch/arm/soc/arm/mps2/*                  @fvincenzo
-arch/arm/soc/atmel_sam/sam4s             @fallrisk
-arch/arm/soc/nxp*/                       @MaureenHelm
-arch/arm/soc/st_stm32/                   @erwango
-arch/arm/soc/st_stm32/stm32f4/*          @rsalveti @idlethread
-arch/arm/soc/ti_simplelink/cc32xx        @GAnthony
-arch/arm/soc/ti_simplelink/msp432p4xx    @Mani-Sadhasivam
+soc/arm/arm/mps2/*                       @fvincenzo
+soc/arm/atmel_sam/sam4s                  @fallrisk
+soc/arm/nxp*/                            @MaureenHelm
+soc/arm/st_stm32/                        @erwango
+soc/arm/st_stm32/stm32f4/*               @rsalveti @idlethread
+soc/arm/ti_simplelink/cc32xx             @GAnthony
+soc/arm/ti_simplelink/msp432p4xx         @Mani-Sadhasivam
 arch/nios2/                              @andrewboie @ramakrishnapallala
 arch/posix/                              @aescolar
```

Comment:
An entry for soc/posix/ at my name would be logical then.
I guess similarly each other soc/{arch}/ would need a dedicated line with the maintainers in arch/{arch}